fdb-relational-core/src/main/java/com/apple/foundationdb/relational/recordlayer/query/SemanticAnalyzer.java

```diff
@@ -668,31 +668,31 @@ public static void validateLimit(@Nonnull Expression expression) {
         Assert.thatUnchecked(underlying instanceof LiteralValue<?>);
         final var value = ((LiteralValue<?>) underlying).getLiteralValue();
         Assert.notNullUnchecked(value, ErrorCode.INVALID_ROW_COUNT_IN_LIMIT_CLAUSE,
-                () -> String.format("limit value out of range [1, %s]", Integer.MAX_VALUE));
+                () -> String.format(Locale.ROOT, "limit value out of range [1, %s]", Integer.MAX_VALUE));
```

Comment:
It's weird that these are using `%s` instead of `%d`, given that it is an integer.
Not really a thing that should be changed in this PR though.

Reply:
Nice catch, I fixed it.